lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -208,8 +212,17 @@ export class AuthorizationCodeClient extends BaseClient {
 
         parameterBuilder.addClientId(this.config.authOptions.clientId);
 
-        // validate the redirectUri (to be a non null value)
-        parameterBuilder.addRedirectUri(request.redirectUri);
+        /*
+         * For hybrid spa flow, there will be a code but no verifier
+         * In this scenario, don't include redirect uri as auth code will not be bound to redirect URI
+         */
+        if (this.hybridSpa) {
```

Comment:
Was hoping to use heuristics (e.g. public client using an auth code with no pkce params), but was told not to. For now, using explicit flag on AuthorizationCodeClient to denote hybrid flow.

Reply:
Another option would be to subclass this class